docs/getting-started-sandbox.md

```diff
@@ -0,0 +1,91 @@
+   ![Sandbox start](/img/docs/sandbox-start.png)
+2. Click **Try API Constructor**.
+   ![API Constructor](/img/docs/api-constructor1.png)
+   On the **API Constructor** page, you have access to all of PayID's capabilities. The **Create User** page opens by default.
+3. Under **API Details**, enter the value for this PayID to create a full PayID address. Example: `starlight$loisrp.sandbox.payid.org`. This is a PayID that you can use to send or receive currency.
+4. Under **Address1**, select the network you want to use. For experimentation, you can choose any of the networks, and you can use a real or fake address for it. This example uses a live address on the XRPL network.
+   - Select **XRPL** for the **Payment Network**
+   - Enter `rUAuauaXNTskaHdsz1TWnHWvB8goNXQaVy` for the **Payment Address**.
+   - The **Destination Tag** can be used to enter a number that is typically used to map to a user in another system. You can experiment by adding a value.
+   - Select **Testnet** or **Mainnet**, depending whether you want to have the address be valid for test currency or real currency.
+   - If you want to associate another address to this PayID, click **+** and repeat the process for another address. Your PayID can have multiple addresses on multiple networks on the same PayID.
+5. By default, **cURL** is selected on the top right. The sandbox constructs the cURL snippet you need to create a user. You can also select **JavaScript** to run a script to create a user.
+6. Click **Run**. Whether you used cURL or JavaScript, the JSON for the newly created user looks the same.
+   ```json
+   {
+     "payId": "starlight$loisrp.sandbox.payid.org",
+     "addresses": [
+       {
+         "paymentNetwork": "XRPL",
+         "details": {
+           "address": "rUAuauaXNTskaHdsz1TWnHWvB8goNXQaVy",
+           "tag": "56"
+         }
+       }
+     ]
+   }
+   ```
+
+## Get Payment Information
+
+**Get Payment Information** is a public API, and does not require your Admin API token. You can use this API for any PayID.
+
+1. Click **Get Payment Information** on the left.
+2. Enter the PayID. On the sandbox, you are restricted to PayIDs on your server. Enter the PayID you just created, or another one you happen to know. Select either **cURL** or **JavaScript** at the top right.
+3. Click **Run**. The JSON response provides information about the PayID.
+
+## Get User
+
+**Get User** provides the same information as **Get Payment Information**, but **Get User** does so through the private Admin API.
+
+1. Click **Get User** on the left.
+2. Enter the PayID you want to look up.
+3. Select either **cURL** or **JavaScript**.
+4. Click **Run**. The JSON response provides information about the PayID.
+
+If you enter a nonexistent PayID, you get the response "Failed to find the PayID <_value_>".
```

Comment:
> "Failed to find the PayID <_value_>"

Trivial again, but consider making this code font if it's part of the response. And if you capitalize the variable, you won't need to use the underscores, too. 

> `Failed to find the PayID <NONEXISTENT_PAYID_VALUE>`

Reply:
Added code font. Kept quotes, because they are part of the response.